### PR TITLE
WPF 890011 - Updated sample codes and images for Gantt UG 

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have updated sample codes and images for Gantt UG content.

### CODE REVIEW

File: wpf/Gantt/custom-node-style.md [hunk 1]
```diff
@@ -246,44 +239,159 @@ The following code illustrates how to define style:
...
+                                           TaskNameMapping="TaskName"
+                                           StartDateMapping="StartDate"
+                                           ChildMapping="Child"
+                                           FinishDateMapping="FinishDate"
+                                           DurationMapping="Duration"
+                                           MileStoneMapping="IsMileStone"
+                                           PredecessorMapping="Predecessor"
+                                           ProgressMapping="Progress" />
+     </syncfusion:GanttControl.TaskAttributeMapping>
     <sync:GanttControl.Resources>
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-node-style.md:256]: Don't use sync - so many times shared same kind of review corrections. Use syncfusion full name.

Change it all places.

File: wpf/Gantt/custom-node-style.md [hunk 2]
```diff
@@ -50,17 +50,10 @@ Milestone</td><td>
...
-{% capture codesnippet1 %}
+{% tabs %}
 {% highlight xaml %}
 
-<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:gantt="http://schemas.syncfusion.com/wpf"
-                    xmlns:chart="clr-namespace:Syncfusion.Windows.Controls.Gantt.Chart;assembly=Syncfusion.Gantt.Wpf" >
-
+    <!-- Resources -->
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-node-style.md:56]: Move this code to GanttControl.Resources

File: wpf/Gantt/custom-node-style.md [hunk 3]
```diff
@@ -246,44 +239,159 @@ The following code illustrates how to define style:
...
+        taskDetails[3].Child[1].Predecessor.Add(new Predecessor() { GanttTaskIndex = 18, GanttTaskRelationship = GanttTaskRelationship.StartToStart });
+        taskDetails[3].Child[2].Predecessor.Add(new Predecessor() { GanttTaskIndex = 18, GanttTaskRelationship = GanttTaskRelationship.StartToStart });
+        taskDetails[3].Child[3].Predecessor.Add(new Predecessor() { GanttTaskIndex = 19, GanttTaskRelationship = GanttTaskRelationship.StartToStart });
+
+        taskDetails[4].Child[1].Predecessor.Add(new Predecessor() { GanttTaskIndex = 25, GanttTaskRelationship = GanttTaskRelationship.StartToStart });
+        taskDetails[4].Child[2].Predecessor.Add(new Predecessor() { GanttTaskIndex = 28, GanttTaskRelationship = GanttTaskRelationship.StartToStart });
+        taskDetails[4].Child[3].Predecessor.Add(new Predecessor() { GanttTaskIndex = 30, GanttTaskRelationship = GanttTaskRelationship.StartToStart });
+        taskDetails[4].Child[4].Predecessor.Add(new Predecessor() { GanttTaskIndex = 27, GanttTaskRelationship = GanttTaskRelationship.StartToStart });
+        return taskDetails;
+
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-node-style.md:389]: Remove unwanted line used in code

File: wpf/Gantt/custom-schedule.md [hunk 4]
```diff
@@ -132,68 +132,330 @@ To Add CustomNumeric Schedule to an application:
...
+         ObservableCollection<GanttScheduleRowInfo> RowInfo = new ObservableCollection<GanttScheduleRowInfo>();
+         RowInfo.Add(new GanttScheduleRowInfo() { CellsPerUnit = 5 });
+         RowInfo.Add(new GanttScheduleRowInfo() { CellsPerUnit = 2, PixelsPerUnit = 30d });
+         return RowInfo;
+     }
+
+     /// <summary>
+     /// Gets the data.
+     /// </summary>
+     /// <returns></returns>
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:363]: Add return value details.

File: wpf/Gantt/custom-schedule.md [hunk 5]
```diff
@@ -132,68 +132,330 @@ To Add CustomNumeric Schedule to an application:
...
+         {
+             this._topCountriesCollection = value;
+         }
+     }
+
+
+     /// <summary>
+     /// Gets the Numeric Schedule Items Info
+     /// </summary>
+     /// <returns></returns>
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:351]: Add return value details

File: wpf/Gantt/custom-schedule.md [hunk 6]
```diff
@@ -132,68 +132,330 @@ To Add CustomNumeric Schedule to an application:
...
+         taskDetails.Add((new TopCountries() { Id = 32, Name = "Uzbekistan", Start = 0, End = 8.5, Rank = 14 }));
+         taskDetails.Add((new TopCountries() { Id = 33, Name = "Yemen", Start = 0, End = 8.016, Rank = 19 }));
+         taskDetails.Add(new TopCountries() { Id = 34, Name = "Zambia", Start = 0, End = 7.601, Rank = 25 });
+         taskDetails.Add((new TopCountries() { Id = 35, Name = "Zimbabwe", Start = 0, End = 9.006, Rank = 11 }));
+
+         return taskDetails;
+     }
+
+     #region Image and Country Collection
+     /// <summary>
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:408]: Create a line to make code readibility

File: wpf/Gantt/custom-schedule.md [hunk 7]
```diff
@@ -132,68 +132,330 @@ To Add CustomNumeric Schedule to an application:
...
+         CountryNamesandFlags.Add("Republic of China(Taiwan)", "Flag_Republic_of_China.png");
+         CountryNamesandFlags.Add("Republic of the Congo", "Flag_Republic_of_the_Congo.png");
+         CountryNamesandFlags.Add("Singapore", "Flag_Singapore.png");
+         CountryNamesandFlags.Add("Sri Lanka", "Flag_Sri_Lanka.png");
+         CountryNamesandFlags.Add("Thailand", "Flag_Thailand.png");
+         CountryNamesandFlags.Add("Turkey", "Flag_Turke.png");
+         CountryNamesandFlags.Add("Turkmenistan", "Flag_Turkmenistan.png");
+         CountryNamesandFlags.Add("Uruguay", "Flag_Uruguay.png");
+         CountryNamesandFlags.Add("Uzbekistan", "Flag_Uzbekistan.png");
+         CountryNamesandFlags.Add("Yemen", "Flag_Yemen.png");
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:445]: rename to `countryFlagDetails`

File: wpf/Gantt/custom-schedule.md [hunk 8]
```diff
@@ -287,94 +558,346 @@ public IList<GanttScheduleRowInfo> GetCustomScheduleSource()
...
+{
+    /// <summary>
+    /// Gets or sets the task collection.
+    /// </summary>
+    public ObservableCollection<Task> TaskCollection { get; set; }
 
-The following image shows Custom DateTime Schedule:
+    public ViewModel()
+    {
+    TaskCollection = this.GetDataSource();
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:825]: Use prefix of `this`

File: wpf/Gantt/custom-schedule.md [hunk 9]
```diff
@@ -287,94 +558,346 @@ public IList<GanttScheduleRowInfo> GetCustomScheduleSource()
...
+    }
 
+    /// <summary>
+    /// Gets the data.
+    /// </summary>
+    /// <returns></returns>
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:831]: Add return type value details.

File: wpf/Gantt/custom-schedule.md [hunk 10]
```diff
@@ -287,94 +558,346 @@ public IList<GanttScheduleRowInfo> GetCustomScheduleSource()
...
+        return taskDetails;
+    }
+}
 
+{% endhighlight  %}
+{% endtabs %}
 
-![Custom-Schedule_images2](Custom-Schedule_images/Custom-Schedule_img2.png)
 
+The following image shows Custom DateTime Schedule:
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:898]: The following image shows custom DateTime schedule:

File: wpf/Gantt/datetime-indicator-customization.md [hunk 11]
```diff
@@ -65,29 +65,51 @@ The following code samples illustrate how to customize the DateTime indicator.
...
+        <local:ViewModel/>
+    </syncfusion:GanttControl.DataContext>
+ </syncfusion:GanttControl>
 
 {% endhighlight  %}
 
 {% highlight c# %}
 
+//Initializing Gantt
+ this.ganttControl.ItemsSource = new ViewModel().TaskCollection;
```
Comment (Karthickmani97) [inline wpf/Gantt/datetime-indicator-customization.md:97]: Remove initializing gantt comment.

File: wpf/Gantt/datetime-indicator-customization.md [hunk 12]
```diff
@@ -100,6 +122,114 @@ this.Gantt.CurrentDateLine.StrokeDashArray = strokeArray;
...
+        set
+        {
+            this.taskCollections = value;
+        }
+    }
+
+    /// <summary>
+    /// Gets the data.
+    /// </summary>
+    /// <returns></returns>
```
Comment (Karthickmani97) [inline wpf/Gantt/datetime-indicator-customization.md:162]: Add return value details.

File: wpf/Gantt/import-and-export-support.md [hunk 13]
```diff
@@ -69,19 +69,42 @@ The following code illustrates how to Import and Export Task Details from or to
...
-            Command="{Binding ImportFromXMLCommand, ElementName=gantt}"
-            Content="Import From XML" />
+<StackPanel Orientation="Horizontal"
+                 HorizontalAlignment="Center">
+         <Button Height="25"
+                 HorizontalAlignment="Center"
+                 VerticalAlignment="Center"
+                 Margin="0,10,0,0"
+                 Width="200"
+                 Click="SaveButton_Click"
```
Comment (Karthickmani97) [inline wpf/Gantt/import-and-export-support.md:79]: Change to `OnSaveButtonClick`

File: wpf/Gantt/import-and-export-support.md [hunk 14]
```diff
@@ -69,19 +69,42 @@ The following code illustrates how to Import and Export Task Details from or to
...
+                 Content="Export To XML" />
+         <Button Height="25"
+                 HorizontalAlignment="Center"
+                 VerticalAlignment="Center"
+                 Margin="0,10,0,0"
+                 Width="200"
+                 Click="Open_Click"
```
Comment (Karthickmani97) [inline wpf/Gantt/import-and-export-support.md:86]: Change to `OnOpenButtonClick`

File: wpf/Gantt/import-and-export-support.md [hunk 15]
```diff
@@ -69,19 +69,42 @@ The following code illustrates how to Import and Export Task Details from or to
...
-<StackPanel Orientation="Horizontal" HorizontalAlignment="Center">
-    <Button Height="25" HorizontalAlignment="Center" VerticalAlignment="Center" 
-            Margin="0,10,0,0" Width="200"
-            Command="{Binding ExportToXMLCommand, ElementName=gantt}" 
-            Content="Export To XML" />
-    <Button Height="25" HorizontalAlignment="Center" VerticalAlignment="Center" 
-            Margin="0,10,0,0" Width="200"
-            Command="{Binding ImportFromXMLCommand, ElementName=gantt}"
-            Content="Import From XML" />
+<StackPanel Orientation="Horizontal"
```
Comment (Karthickmani97) [inline wpf/Gantt/import-and-export-support.md:72]: Add it in grid and add 2 rows and handle these codes.

File: wpf/Gantt/resource-view-gantt-inline-items.md [hunk 16]
```diff
@@ -53,60 +53,591 @@ To populate a Resource view Gantt in a Custom Numeric Schedule:
...
-                    ItemsSource="{Binding TeamDetails}">
-    <gantt:GanttControl.TaskAttributeMapping>
-        <gantt:TaskAttributeMapping TaskNameMapping="Name"
-                                    ChildMapping="SubItems"
-                                    StartPointMapping="Start"
-                                    FinishPointMapping="End"
-                                    InLineTaskMapping="InLineItems"/>
-    </gantt:GanttControl.TaskAttributeMapping>
-</gantt:GanttControl>
+<!-- Resource -->
```
Comment (Karthickmani97) [inline wpf/Gantt/resource-view-gantt-inline-items.md:59]: Move under resource

File: wpf/Gantt/resource-view-gantt-inline-items.md [hunk 17]
```diff
@@ -119,77 +650,627 @@ To populate the Resource view Gantt with a date-time schedule:
...
-                     ChildMapping="SubItems"   
-                      FinishDateMapping="FinishDate"   
-                     InLineTaskMapping="InLineItems">
-
-                    </gantt:TaskAttributeMapping>
-                </gantt:GanttControl.TaskAttributeMapping>
- </gantt:GanttControl>
-
+<!-- Resources -->
+<!-- Header Node style-->
```
Comment (Karthickmani97) [inline wpf/Gantt/resource-view-gantt-inline-items.md:657]: Move to under reources

File: wpf/Gantt/resource-view-gantt-inline-items.md [hunk 18]
```diff
@@ -119,77 +650,627 @@ To populate the Resource view Gantt with a date-time schedule:
...
+         {
+             foreach (Item item in e.NewItems)
+             {
+                 item.PropertyChanged += ItemPropertyChanged;
+             }
+         }
+         else
+         {
+             foreach (Item item in e.OldItems)
+                 item.PropertyChanged -= ItemPropertyChanged;
```
Comment (Karthickmani97) [inline wpf/Gantt/resource-view-gantt-inline-items.md:1086]: add brackets in between codes.

File: wpf/Gantt/resource-view-gantt-inline-items.md [hunk 19]
```diff
@@ -119,77 +650,627 @@ To populate the Resource view Gantt with a date-time schedule:
...
+         }
+         UpdateDates();
```
Comment (Karthickmani97) [inline wpf/Gantt/resource-view-gantt-inline-items.md:1088]: Add new line and use prefix `this`

File: wpf/Gantt/resource-view-gantt-inline-items.md [hunk 20]
```diff
@@ -119,77 +650,627 @@ To populate the Resource view Gantt with a date-time schedule:
...
+    /// </summary>
+    /// <returns></returns>
+    public ObservableCollection<Item> GetTeamInfo()
+    {
+        DateTime dtS = DateTime.Today;
+
+        ObservableCollection<Item> teams = new ObservableCollection<Item>();
+
+        teams.Add(new Item() { Name = "RDU Team" });
+        Item Person = new Item() { Name = "Robert" };
```
Comment (Karthickmani97) [inline wpf/Gantt/resource-view-gantt-inline-items.md:1177]: rename to `person`

File: wpf/Gantt/resource-view-gantt-inline-items.md [hunk 21]
```diff
@@ -119,77 +650,627 @@ To populate the Resource view Gantt with a date-time schedule:
...
+    }
+
+
+    /// <summary>
+    /// Gets the team info.
+    /// </summary>
+    /// <returns></returns>
+    public ObservableCollection<Item> GetTeamInfo()
+    {
+        DateTime dtS = DateTime.Today;
```
Comment (Karthickmani97) [inline wpf/Gantt/resource-view-gantt-inline-items.md:1172]: chage to `dateTime`

File: wpf/Gantt/zooming.md [hunk 22]
```diff
@@ -33,36 +33,172 @@ To add the built-in zooming:
 {% tabs %}
 {% highlight xaml %}
 
-<Slider Minimum="80" Maximum="600" Value="100" x:Name="ZoomSlider" Width="150"/>
-
-<sync:GanttControl  x:Name="Gantt" 
-                    ItemsSource="{Binding GanttItemSource}" 
-                    UseOnDemandSchedule="True"
-                    ZoomFactor="{Binding ElementName=ZoomSlider, Path=Value}"/>
+<Slider x:Name="ZoomSlider" Minimum="80" Maximum="600" Value="100" Width="150" ValueChanged="slider_ValueChanged"/>
```
Comment (Karthickmani97) [inline wpf/Gantt/zooming.md:36]: use grid, add handle slider and gantt.

File: wpf/Gantt/zooming.md [hunk 23]
```diff
@@ -33,36 +33,172 @@ To add the built-in zooming:
...
+slider.Minimum = 1;
+slider.Minimum = 600;
+
+//Hooking the value changed event of the slider
+slider.ValueChanged += slider_ValueChanged;
+
+/// <summary>
+/// Handles the ValueChanged event of the slider control.
+/// </summary>
+void slider_ValueChanged(object sender, RoutedPropertyChangedEventArgs<double> e)
```
Comment (Karthickmani97) [inline wpf/Gantt/zooming.md:91]: rename to `OnSliderValueChanged`

File: wpf/Gantt/zooming.md [hunk 24]
```diff
@@ -113,16 +249,48 @@ To add custom zooming:
...
-<sync:GanttControl x:Name="Gantt" 
-                   ItemsSource="{Binding GanttItemSource}" 
-                   UseOnDemandSchedule="True"
-                   ZoomFactor="{Binding ZoomFactor}"
-                   ZoomChanged="Gantt_ZoomChanged"/>
+ <syncfusion:GanttControl x:Name="ganttControl"
+                          ItemsSource="{Binding TaskCollection}" 
+                          UseOnDemandSchedule="True"
+                          ZoomFactor="{Binding ZoomFactor}"
+                          ZoomChanged="Gantt_ZoomChanged">
```
Comment (Karthickmani97) [inline wpf/Gantt/zooming.md:256]: rename to `OnGanttZoomChanged`

File: wpf/Gantt/zooming.md [hunk 25]
```diff
@@ -230,6 +398,114 @@ private void Gantt_ZoomChanged(object sender, ZoomChangedEventArgs args)
...
+
+        taskDetails.Add(new TaskDetails() { TaskId = 8, TaskName = "Production", StartDate = new DateTime(2011, 7, 3), FinishDate = new DateTime(2011, 7, 14), Progress = 40d });
+        taskDetails[1].Child.Add((new TaskDetails() { TaskId = 9, TaskName = "Software Components", StartDate = new DateTime(2011, 7, 3), FinishDate = new DateTime(2011, 7, 5), Progress = 20d }));
+        taskDetails[1].Child.Add((new TaskDetails() { TaskId = 10, TaskName = "Localization Component - User Interface", StartDate = new DateTime(2011, 7, 6), FinishDate = new DateTime(2011, 7, 7), Progress = 20d }));
+        taskDetails[1].Child.Add((new TaskDetails() { TaskId = 11, TaskName = "User Assistance Components", StartDate = new DateTime(2011, 7, 10), FinishDate = new DateTime(2011, 7, 14), Progress = 10d }));
+        taskDetails[1].Child.Add((new TaskDetails() { TaskId = 12, TaskName = "Software components complete", StartDate = new DateTime(2011, 7, 14), FinishDate = new DateTime(2011, 7, 19), Progress = 10d }));
+
+
```
Comment (Karthickmani97) [inline wpf/Gantt/zooming.md:457]: Removed unwanted added.

**Review by Karthickmani97 — COMMENTED** [review]
@TamilarasanGunasekaran , Address my review corrections.

File: wpf/Gantt/custom-node-style.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-node-style.md:56]: Moved.

File: wpf/Gantt/custom-node-style.md (hunk 3 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-node-style.md:389]: Removed.

File: wpf/Gantt/custom-schedule.md (hunk 4 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-schedule.md:363]: Added.

File: wpf/Gantt/custom-schedule.md (hunk 5 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-schedule.md:351]: Added.

File: wpf/Gantt/custom-schedule.md (hunk 6 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-schedule.md:408]: Created.

File: wpf/Gantt/custom-schedule.md (hunk 7 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-schedule.md:445]: Renamed.

File: wpf/Gantt/custom-schedule.md (hunk 8 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-schedule.md:825]: Updated.

File: wpf/Gantt/custom-schedule.md (hunk 9 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-schedule.md:831]: Added.

File: wpf/Gantt/custom-schedule.md (hunk 10 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-schedule.md:898]: Updated.

File: wpf/Gantt/datetime-indicator-customization.md (hunk 11 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/datetime-indicator-customization.md:97]: Removed.

File: wpf/Gantt/datetime-indicator-customization.md (hunk 12 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/datetime-indicator-customization.md:162]: Added.

File: wpf/Gantt/import-and-export-support.md (hunk 13 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/import-and-export-support.md:79]: Changed.

File: wpf/Gantt/import-and-export-support.md (hunk 14 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/import-and-export-support.md:86]: Changed.

File: wpf/Gantt/import-and-export-support.md (hunk 15 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/import-and-export-support.md:72]: Added.

File: wpf/Gantt/resource-view-gantt-inline-items.md (hunk 16 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/resource-view-gantt-inline-items.md:59]: Moved.

File: wpf/Gantt/resource-view-gantt-inline-items.md (hunk 17 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/resource-view-gantt-inline-items.md:657]: Moved.

File: wpf/Gantt/resource-view-gantt-inline-items.md (hunk 18 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/resource-view-gantt-inline-items.md:1086]: Added.

File: wpf/Gantt/resource-view-gantt-inline-items.md (hunk 19 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/resource-view-gantt-inline-items.md:1088]: Added.

File: wpf/Gantt/resource-view-gantt-inline-items.md (hunk 20 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/resource-view-gantt-inline-items.md:1177]: Renamed.

File: wpf/Gantt/resource-view-gantt-inline-items.md (hunk 21 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/resource-view-gantt-inline-items.md:1172]: Changed.

File: wpf/Gantt/zooming.md (hunk 22 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/zooming.md:36]: Updated.

File: wpf/Gantt/zooming.md (hunk 24 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/zooming.md:256]: Renamed.

File: wpf/Gantt/zooming.md (hunk 25 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/zooming.md:457]: Removed

File: wpf/Gantt/custom-node-style.md [hunk 26]
```diff
@@ -50,252 +50,401 @@ Milestone</td><td>
...
+                                           TaskNameMapping="TaskName"
+                                           StartDateMapping="StartDate"
+                                           ChildMapping="Child"
+                                           FinishDateMapping="FinishDate"
+                                           DurationMapping="Duration"
+                                           MileStoneMapping="IsMileStone"
+                                           PredecessorMapping="Predecessor"
+                                           ProgressMapping="Progress" />
+     </syncfusion:GanttControl.TaskAttributeMapping>
+    <syncfusion:GanttControl.Resources>
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-node-style.md:69]: <syncfusion:GanttControl.Resources> , added two times. remove it

File: wpf/Gantt/custom-node-style.md [hunk 27]
```diff
@@ -50,252 +50,401 @@ Milestone</td><td>
...
+        return taskDetails;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![gantt-custom-node-style](Custom-Node-Style_images/gantt-custom-node-style.png)
 
 Custom Node Style
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-node-style.md:449]: Add image for custom node style.

File: wpf/Gantt/custom-schedule.md [hunk 28]
```diff
@@ -132,68 +132,328 @@ To Add CustomNumeric Schedule to an application:
...
+taskAttributeMapping.FinishDateMapping = "End";
+taskAttributeMapping.ProgressMapping = "Complete";
+taskAttributeMapping.ResourceInfoMapping = "Resource";
+this.ganttControl.TaskAttributeMapping = taskAttributeMapping;
 
-return RowInfo;
+{% endhighlight  %}
 
-}   
+{% highlight c# tabtitle="ViewModel.cs" %}
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:286]: Add `TopCountries` model file.

File: wpf/Gantt/custom-schedule.md [hunk 29]
```diff
@@ -132,68 +132,328 @@ To Add CustomNumeric Schedule to an application:
...
+         <Setter.Value>
+             <ControlTemplate TargetType="{x:Type gantt:GanttNode}">
+                 <Border
+                     Name="PART_Border"
+                     Height="21"
+                     VerticalAlignment="Center"
+                     BorderThickness="0.5"
+                     Opacity="0.8">
+                     <interact:Interaction.Behaviors>
+                         <local:NumericGanttNodeCustomizationBehavior />
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:166]: `NumericGanttNodeCustomizationBehavior` is this need?

File: wpf/Gantt/custom-schedule.md [hunk 30]
```diff
@@ -222,50 +482,62 @@ The following code illustrates this:
...
+taskAttributeMapping.PredecessorMapping = "Predecessor";
+taskAttributeMapping.ResourceInfoMapping = "Resource";
+this.ganttControl.TaskAttributeMapping = taskAttributeMapping;
 
-this.Gantt.CustomScheduleSource = this.GetCustomScheduleSource();
+// Assigning the custom schedule Items Source.
+this.ganttControl.CustomScheduleSource = this.GetCustomScheduleSource();
 
 // Hooks the schedule cell created event to customize the schedule cell appearance.
+this.ganttControl.ScheduleCellCreated += this.OnGanttScheduleCellCreated;
```
Comment (Karthickmani97) [inline wpf/Gantt/custom-schedule.md:532]: Is this event method `OnGanttScheduleCellCreated`, where it is declared?

File: wpf/Gantt/highlighting-tasks.md [hunk 31]
```diff
@@ -52,70 +52,958 @@ To highlight a set of tasks in Gantt chart region:
 {% tabs %}
 {% highlight xaml %}
 
-<sync:GanttControl x:Name="Gantt"
-                   ItemsSource="{Binding GanttItemSource}" 
-                   HighlightedItems="{Binding HighlightedTasks}">
-</sync:GanttControl>
+ <syncfusion:GanttControl x:Name="ganttControl"
+                          HighlightItemBrush="{Binding Path=Brush, ElementName=HighlightBrush, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged"
```
Comment (Karthickmani97) [inline wpf/Gantt/highlighting-tasks.md:56]: Don't copy code directly from Gantt SB. change code based on req.

File: wpf/Gantt/import-and-export-support.md [hunk 32]
```diff
@@ -108,17 +140,136 @@ private void Open_Click(object sender, System.Windows.RoutedEventArgs e)
...
+
+        taskDetails.Add(new TaskDetails() { TaskId = 8, TaskName = "Production", StartDate = new DateTime(2011, 7, 3), FinishDate = new DateTime(2011, 7, 14), Progress = 40d });
+        taskDetails[1].Child.Add((new TaskDetails() { TaskId = 9, TaskName = "Software Components", StartDate = new DateTime(2011, 7, 3), FinishDate = new DateTime(2011, 7, 5), Progress = 20d }));
+        taskDetails[1].Child.Add((new TaskDetails() { TaskId = 10, TaskName = "Localization Component - User Interface", StartDate = new DateTime(2011, 7, 6), FinishDate = new DateTime(2011, 7, 7), Progress = 20d }));
+        taskDetails[1].Child.Add((new TaskDetails() { TaskId = 11, TaskName = "User Assistance Components", StartDate = new DateTime(2011, 7, 10), FinishDate = new DateTime(2011, 7, 14), Progress = 10d }));
+        taskDetails[1].Child.Add((new TaskDetails() { TaskId = 12, TaskName = "Software components complete", StartDate = new DateTime(2011, 7, 14), FinishDate = new DateTime(2011, 7, 19), Progress = 10d }));
+
+
```
Comment (Karthickmani97) [inline wpf/Gantt/import-and-export-support.md:217]: Remove unwanted lines.

File: wpf/Gantt/strip-lines.md [hunk 33]
```diff
@@ -229,10 +229,51 @@ Event </td></tr>
...
+taskAttributeMapping.ChildMapping = "Child";
+taskAttributeMapping.FinishDateMapping = "FinishDate";
+taskAttributeMapping.DurationMapping = "Duration";
+taskAttributeMapping.MileStoneMapping = "IsMileStone";
+taskAttributeMapping.PredecessorMapping = "Predecessor";
+taskAttributeMapping.ProgressMapping = "Progress";
+taskAttributeMapping.ResourceInfoMapping = "Resource";
+this.ganttControl.TaskAttributeMapping = taskAttributeMapping;
 
 StripCollection =  new List<StripLineInfo>();
```
Comment (Karthickmani97) [inline wpf/Gantt/strip-lines.md:278]: Where `StripCollection` varaible is declared.

File: wpf/Gantt/strip-lines.md [hunk 34]
```diff
@@ -258,49 +298,172 @@ private List<StripLineInfo> GetStripCollection()
...
```
Comment (Karthickmani97) [inline wpf/Gantt/strip-lines.md:468]: `StripCollection` varable not declared.

File: wpf/Gantt/zooming.md [hunk 35]
```diff
@@ -33,36 +33,193 @@ To add the built-in zooming:
...
+taskAttributeMapping.ResourceInfoMapping = "Resource";
+this.ganttControl.TaskAttributeMapping = taskAttributeMapping;
+
+/// Defining the Slider
+Slider slider = new Slider();
+slider.Minimum = 1;
+slider.Minimum = 600;
+
+//Hooking the value changed event of the slider
+slider.ValueChanged += slider_ValueChanged;
```
Comment (Karthickmani97) [inline wpf/Gantt/zooming.md:108]: slider_ValueChanged , where it is used?

Comment (TamilarasanGunasekaran) [reply wpf/Gantt/zooming.md:108]: Event name Updated.

File: wpf/Gantt/strip-lines.md (hunk 34 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/strip-lines.md:468]: Declared.

File: wpf/Gantt/strip-lines.md (hunk 33 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/strip-lines.md:278]: Declared.

File: wpf/Gantt/import-and-export-support.md (hunk 32 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/import-and-export-support.md:217]: Removed.

File: wpf/Gantt/custom-schedule.md (hunk 29 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-schedule.md:166]: No need, removed.

File: wpf/Gantt/custom-schedule.md (hunk 28 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-schedule.md:286]: Model file added.

File: wpf/Gantt/custom-node-style.md (hunk 26 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-node-style.md:69]: Removed.

File: wpf/Gantt/highlighting-tasks.md (hunk 31 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/highlighting-tasks.md:56]: Changed.

File: wpf/Gantt/custom-node-style.md (hunk 27 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/custom-node-style.md:452]: Added.

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

**Pull request merged**